google/resource_runtimeconfig_config.go

```diff
@@ -0,0 +1,148 @@
+package google
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/runtimeconfig/v1beta1"
+)
+
+var runtimeConfigFullName *regexp.Regexp
+
+func init() {
+	runtimeConfigFullName = regexp.MustCompile("^projects/([^/]+)/configs/(.+)$")
```

Comment:
Didn't know you could do this; for some reason I thought only primitives could be initialized in this manner (I guess this works because it's a var rather than a constant?)